jena-core/src/main/java/org/apache/jena/vocabulary/OWL1.java

```diff
@@ -0,0 +1,332 @@
+
+    public static final Property incompatibleWith = Init.incompatibleWith();
+
+    public static final Property minCardinality = Init.minCardinality();
+
+    public static final Property complementOf = Init.complementOf();
+
+    public static final Property onProperty = Init.onProperty();
+
+    public static final Property equivalentProperty = Init.equivalentProperty();
+
+    public static final Property inverseOf = Init.inverseOf();
+
+    public static final Property backwardCompatibleWith = Init.backwardCompatibleWith();
+
+    public static final Property differentFrom = Init.differentFrom();
+
+    public static final Property priorVersion = Init.priorVersion();
+
+    public static final Property imports = Init.imports();
+
+    public static final Property allValuesFrom = Init.allValuesFrom();
+
+    public static final Property unionOf = Init.unionOf();
+
+    public static final Property hasValue = Init.hasValue();
+
+    public static final Property someValuesFrom = Init.someValuesFrom();
+
+    public static final Property disjointWith = Init.disjointWith();
+
+    public static final Property cardinality = Init.cardinality();
+
+    public static final Property intersectionOf = Init.intersectionOf();
+
+    // Vocabulary classes
+    ///////////////////////////
+
+    public static final Resource Thing = Init.Thing();
+
+    public static final Resource DataRange = Init.DataRange();
+
+    public static final Resource Ontology = Init.Ontology();
+
+    public static final Resource DeprecatedClass = Init.DeprecatedClass();
+
+    public static final Resource AllDifferent = Init.AllDifferent();
+
+    public static final Resource DatatypeProperty = Init.DatatypeProperty();
+
+    public static final Resource SymmetricProperty = Init.SymmetricProperty();
+
+    public static final Resource TransitiveProperty = Init.TransitiveProperty();
+
+    public static final Resource DeprecatedProperty = Init.DeprecatedProperty();
+
+    public static final Resource AnnotationProperty = Init.AnnotationProperty();
+
+    public static final Resource Restriction = Init.Restriction();
+
+    public static final Resource Class = Init.Class();
+
+    public static final Resource OntologyProperty = Init.OntologyProperty();
+
+    public static final Resource ObjectProperty = Init.ObjectProperty();
+
+    public static final Resource FunctionalProperty = Init.FunctionalProperty();
+
+    public static final Resource InverseFunctionalProperty = Init.InverseFunctionalProperty();
+
+    public static final Resource Nothing = Init.Nothing();
+
+    // Vocabulary individuals
+    ///////////////////////////
+
+    /**
+     * OWL constants are used during Jena initialization.
+     * <p>
+     * If that initialization is triggered by touching the OWL class,
+     * then the constants are null.
+     * <p>
+     * So for these cases, call this helper class: Init.function()
+     */
+    public static class Init {
```

Comment:
This pattern (Init inner class) should be unnecessary because initialization is more controlled by `ServiceLoader` and `InitJenaCore`.

I tried making `Init`, class and functions, all `private static` and it didn't seem to cause any problems.

Maybe it would be a good idea to either make this change or at least deprecate all uses of `Init.*` so any user code using them is warned.

Reply:
made a separate commit with fixed OWL1, RDF, XSD vocabularies